web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -71,17 +72,22 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
     inactive: true,
   });
   const [showAnnotations, setShowAnnotations] = useLocalStorage('alerts-annotations-status', { checked: false });
-
   const toggleFilter = (ruleState: RuleState) => () => {
     setFilter({
       ...filter,
       [ruleState]: !filter[ruleState],
     });
   };
 
-  const handleSearchChange = (e: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
-    if (e.target.value !== '') {
-      const pattern = e.target.value.trim();
+  const updateURL = (expr: string): void => {
```

Comment:
Let's still rename `expr` to `search` here and in the other places.

Reply:
ready! :)